cxplat/cxplat_test/cxplat_fault_injection_test.cpp

```diff
@@ -21,11 +20,11 @@ struct _on_exit
 
 enum class _fault_injection_test_state
 {
-    ExpectedFault,               /// < Fault should be injected.
-    DontExpectedFault,            /// < Fault should not be injected.
-    DontExpectedFaultAfterReload, /// < Fault should not be injected after reloading.
-    ExpectedFaultAfterReset,     /// < Fault should be injected after reset.
-    ExpectedFaultDifferentCallsite,   /// < Fault should be injected at a different callsite.
+    ExpectedFault,                  /// < Fault should be injected.
+    DontExpectedFault,              /// < Fault should not be injected.
+    DontExpectedFaultAfterReload,   /// < Fault should not be injected after reloading.
```

Comment:
I take it back, "don't expected" doesn't read right.  Either "InjectFault",  "DontInjectFault", etc., or change back to Not and update the text descriptions to use "expected" and not "injected" with some appropriate wording.